docs/samples/edr-api-overview/edr-api-overview.md

```diff
@@ -133,3 +169,85 @@ This endpoint, through the `transfer-process-id` passed as `path variable`, will
 ```
 
 > Please note that now with the EDR you are able to request the `Asset` data from provider's `data-plane`.
+
+### EDR Management | Deleting a cached EDR
+
+This endpoint will delete the EDR entry associated with the `transfer-process-id` and it will remove the EDR itself
+from the vault.
+
+| Path                                          | Method | Query Params             |
```

Comment:
I would just link to the published openapi spec, all documentation should be there